homeassistant/components/remote/__init__.py

```diff
@@ -103,6 +104,7 @@ async def async_setup(hass: HomeAssistantType, config: ConfigType) -> bool:
         {
             vol.Optional(ATTR_DEVICE): cv.string,
             vol.Optional(ATTR_COMMAND): vol.All(cv.ensure_list, [cv.string]),
+            vol.Optional(ATTR_COMMAND_TYPE): cv.string,
```

Comment:
This changed the entity model without approval in an architecture PR. We can't do that.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
I agree, this needs to be reverted. Opened up a PR do so in #40486